plugins/CoreUpdater/Controller.php

```diff
@@ -172,6 +173,17 @@ public function oneClickUpdate()
         return $view->render();
     }
 
+    public function oneClickUpdatePartTwo()
+    {
+        Piwik::checkUserHasSuperUserAccess();
```

Comment:
Wondering what happens if user was just logged out in between this step and the previous?  Eg maybe because of what we noticed in https://github.com/matomo-org/matomo/pull/15755/files#r405990135 ?

I guess otherwise it's pretty much unlikely that a cookie would expire just between those two requests as we extend the cookie on every request by 30 min AFAIK and previous step shouldn't take more than 30min.

I suppose we would notice such an issue in the one click updater test so things should be all good 👍

Reply:
Definitely want to test w/ the release candidate, regardless.